### PR TITLE
Add generateAutoloads composer command

### PR DESCRIPTION
**Pull request by blankse**
With this change it is possible to generate the legacy autoloads after composer update/install.

### CODE REVIEW

**Comment by andrerom** [comment]
nice, +1 *(backport would make sense also)*

**Comment by blankse** [comment]
Ping @bdunogier @lolautruche

**Comment by yannickroger** [comment]
Are the test supposed to fail ? @bdunogier @andrerom

**Comment by blankse** [comment]
The tests failed the last 4 times: https://travis-ci.org/ezsystems/LegacyBridge/builds

File: bundle/Composer/ScriptHandler.php [hunk 1]
```diff
@@ -71,4 +71,18 @@ public static function installLegacyBundlesExtensions(CommandEvent $event)
...
         static::executeCommand($event, $appDir, 'ezpublish:legacybundles:install_extensions ' . $symlink);
     }
+
+    public static function generateAutoloads(CommandEvent $event)
+    {
+        $options = self::getOptions($event);
+        $appDir = $options['symfony-app-dir'];
+
+        if (!is_dir($appDir)) {
+            echo 'The symfony-app-dir (' . $appDir . ') specified in composer.json was not found in ' . getcwd() . ', can not install assets.' . PHP_EOL;
```
Comment (emodric) [inline bundle/Composer/ScriptHandler.php]: Error message probably needs changing :)

Comment (blankse) [inline bundle/Composer/ScriptHandler.php]: I have corrected it :)

**Comment by andrerom** [comment]
> Are the test supposed to fail ?

Obviously not if you click on the link ;)
see #42

**Comment by emodric** [comment]
Some of them can be fixed, but with PR to eZ kernel (private property visibility needs to be changed to protected), but there's one that fails and I'm not really sure what the test does in the first place :D

**Comment by emodric** [comment]
@andrerom 

https://github.com/ezsystems/ezpublish-kernel/pull/1506
https://github.com/ezsystems/LegacyBridge/pull/43

**Comment by blankse** [comment]
Rebased to Master

**Comment by emodric** [comment]
Might I suggest you also add a second command, `generateKernelOverrideAutoloads` which passes `-o` to the autload generation script? :)

**Comment by blankse** [comment]
@emodric I did it :wink:

**Comment by emodric** [comment]
:+1:

**Comment by blankse** [comment]
ping @andrerom @bdunogier @lolautruche

**Comment by emodric** [comment]
Ping :)

**Comment by andrerom** [comment]
we are far from done with the release yet, give us a week or two and we'll be able to look at things outside the release again.

**Comment by emodric** [comment]
@andrerom Your comment probably should've gone to the issue, but okay, I get it.

**Comment by andrerom** [comment]
issue? there is none on this one, hint :)

**Comment by emodric** [comment]
This one :)

https://github.com/ezsystems/LegacyBridge/issues/47

**Comment by emodric** [comment]
Both of the commands are tested and work fine :+1:

**Comment by emodric** [comment]
\o/

Thanks @blankse and @bdunogier !

**Pull request merged**